### PR TITLE
gc-2832 update ranged copyright statements

### PR DESCRIPTION
**Pull request by javadev001001**
Updated ranged copyright statements

### CODE REVIEW

**Review by GCHQDev404 — CHANGES_REQUESTED** [review]
ctrl-R is not a suitable solution.

File: core/access/src/main/java/uk/gov/gchq/gaffer/access/predicate/AccessPredicate.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Crown Copyright
+ * Copyright 2021 Crown Copyright
```
Comment (GCHQDev404) [inline core/access/src/main/java/uk/gov/gchq/gaffer/access/predicate/AccessPredicate.java:2]: This class was not made in 2021.

**Comment by hj325488** [comment]
Is the correct format for the copyright strings: "Copyright <year file created>-<year last modified> Crown Copyright"?

If so, I've written a small helper script that creates this information from the Git history. If this ticket isn't being worked on by anyone else at the moment, I could pick it up and finish it off.

**Comment by GCHQDev404** [comment]
@hj325488 
`Copyright [YYYY] Crown Copyright` first year
`"Copyright [First Year]-[Current Year] Crown Copyright"?` for following years

**Comment by GCHQDev404** [comment]
This PR has gone dormant and will be closed.

**Comment by GCHQDev404** [comment]
Dormant & incomplete.

**Comment by GCHQDev404** [comment]
incorrect branch and issue labelling should be gh-2382 not gc-2832

**Pull request closed without merging**